semantics/src/main/scala/aqua/semantics/Semantics.scala

```diff
@@ -23,4 +24,15 @@ trait Semantics[S[_], C] {
     ast: Ast[S],
     init: C
   ): ProcessResult
+
+  def stateToResult(state: CompilerState[S], ctx: C): ProcessResult =
```

Comment:
Can we make it method of the `CompilerState` or `object Semantics`? It is strange for it to be a method of semantics trait, it is connected to it only by return type